### PR TITLE
Also export (results) as json

### PR DESCRIPTION
**Pull request by waghanza**
Hi,

This `PR` export results as json, file `data.json`.

I also update results in here.

###### Frameworks to rerun :
+ [x] roda
+ [x] luminus
+ [x] laravel
+ [x] driftphp
+ [x] quarkus

Regards,

/cc @SuspiciousLookingOwl

### CODE REVIEW

**Comment by SuspiciousLookingOwl** [comment]
I'm slightly concern about the file size.
```rake
File.open('data.json','w').write(JSON.pretty_generate(data))
```
I assume it will save in prettified JSON. Can you provide one more file (e.g. `data.min.json`) for the minified version?

**Comment by waghanza** [comment]
Good catch @SuspiciousLookingOwl, `data.json` is huge (1.5 M) whereas minified version, data.min.json could be parse easily (1000K).

I can also generate bson or whatever format is suitable for you

**Comment by SuspiciousLookingOwl** [comment]
Seems like the `.min.json` isn't a proper JSON format 👀 
```
{
    :metrics=>[
        {
            :level=>64
        ...
```

**Comment by waghanza** [comment]
> Seems like the `.min.json` isn't a proper JSON format 👀
> 
> ```
> {
>     :metrics=>[
>         {
>             :level=>64
>         ...
> ```

yes

will be so ... missing an other framework to undraft this PR

**Comment by waghanza** [comment]
Alls good for you @SuspiciousLookingOwl ?

**Comment by SuspiciousLookingOwl** [comment]
Hey @waghanza, can you add one more key (`"updatedAt"`) to the json 😅
```json
{
    "updatedAt": "2021-04-13", <-- this
    "metrics": [...],
    "languages": [...]
}
```

**Comment by waghanza** [comment]
done


I think we also could add a `hardware` section, but could be done in an other `PR`

**Comment by SuspiciousLookingOwl** [comment]
Framework name seems to be missing from `frameworks`

Result:
```json
{
  "frameworks": [
    {
      "id": 1,
      "version": 11.6,
      "language": "go",
      "website": "github.com/savsgio/atreugo/blob/master/docs/README.md"
    }
  ]
}

```

Expected:
```json
{
  "frameworks": [
    {
      "id": 1,
      "version": 11.6,
      "language": "go",
      "label": "Atreugo", // missing
      "website": "github.com/savsgio/atreugo/blob/master/docs/README.md"
    }
  ]
}
```

File: .tasks/db.rake [hunk 1]
```diff
@@ -104,4 +104,47 @@ namespace :db do
...
+                  else
+                    (framework_config['framework']['website']).to_s
+                  end
+        unless data[:frameworks].map{|row|row[:id]}.to_a.include?(framework_id)
+          data[:frameworks] << {
+            id: framework_id,
+            version: framework_config.dig('framework', 'version'),
+            language: language,
+            website: website
+          }
```
Comment (waghanza) [inline .tasks/db.rake:134]: yeah @SuspiciousLookingOwl, missing label in here

Comment (waghanza) [reply .tasks/db.rake]: done @SuspiciousLookingOwl

**Pull request merged**